runfullday.sh

```diff
@@ -82,6 +93,13 @@ case $NIC_SPD in
 *) NIC_SPD=10 ;;
 esac
 
+unset MAX_STREAMS
```

Comment:
Please change this to a stream list, with the default list "10, 1000, 10000, 100000, 1000000"